app/model/ProgressStatuses.scala

```diff
@@ -47,4 +47,40 @@ object ProgressStatuses {
   val AssessmentCentreFailedProgress = ApplicationStatuses.AssessmentCentreFailed.toLowerCase()
   val AssessmentCentrePassedNotifiedProgress = ApplicationStatuses.AssessmentCentrePassedNotified.toLowerCase()
   val AssessmentCentreFailedNotifiedProgress = ApplicationStatuses.AssessmentCentreFailedNotified.toLowerCase()
+
+  sealed abstract class ProgressStatus(applicationStatus: ApplicationStatus)
+
+  object ProgressStatus {
+    implicit val progressStatusFormat = new Format[ProgressStatus] {
+      def reads(json: JsValue) = JsSuccess(nameToProgressStatus(json.as[String]))
+      def writes(myEnum: ProgressStatus) = JsString(myEnum.toString)
+    }
+
+    implicit object BSONEnumHandler extends BSONHandler[BSONString, ProgressStatus] {
+      def read(doc: BSONString) = nameToProgressStatus(doc.value)
+      def write(stats: ProgressStatus) = BSON.write(stats.toString)
+    }
+  }
+
+  case object PHASE1_TESTS_INVITED extends ProgressStatus(ApplicationStatus.PHASE1_TESTS)
```

Comment:
We need to make sure that everybody agree with the new capital case convention for progress statuses + we need refactor the old ones